.github/workflows/base-wrapper.yml

```diff
@@ -65,7 +65,7 @@ on:
 
 jobs:
   delegate:
-    uses: graalvm/mandrel/.github/workflows/base.yml@default
+    uses: zakkak/mandrel/.github/workflows/base.yml@2023-09-14-fix-570
```

Comment:
I take it this will be reverted once CI completes?

Reply:
Yes, I better not forget it :)